classes/blocks/class-covers.php

```diff
@@ -364,7 +364,7 @@ private static function populate_posts_for_campaigns( &$fields ) {
 
 			if ( ! empty( $attachment_id ) ) {
 				$tag_remapped['image']    = wp_get_attachment_image_src( $attachment_id, 'medium_large' );
-				$tag_remapped['src_set']  = wp_get_attachment_image_srcset( $attachment_id, 'medium_large' );
+				$tag_remapped['src_set']  = wp_get_attachment_image_srcset( $attachment_id, 'medium_large' ) ?? 'false';
```

Comment:
The `??` operator will only use the second operand if the first one is `null`. In this case the function never returns null, but a bool instead. Even if it did, why would this need to be the string `'false'`? :thinking:

Reply:
I was getting console warnings in the frontend, that `srcSet` was set to a boolean value and needed a string instead

Reply:
`'false'` is not really better because React also does not know what to do with it for the `srcset` prop.